karrot/userauth/api.py

```diff
@@ -115,6 +119,7 @@ def post(self, request):
 
 class ResendMailVerificationCodeView(views.APIView):
     permission_classes = (IsAuthenticated, MailIsNotVerified)
+    serializer_class = inline_serializer(name='', fields={})  # for OpenAPI generation with drf-spectacular
```

Comment:
Maybe as the `inline_serializer(name='', fields={})` snippet is used three times, it could be extracted and given a meaningful name? Reflecting that it's basically a "empty serializer" as I understand it?

Reply:
Funnily, we already have a proper `EmptySerializer` somewhere in our code, maybe that one can be used instead.